packages/itwin/tree-widget/package.json

```diff
@@ -37,36 +37,40 @@
     "rebuild": "npm run clean && npm run build"
   },
   "dependencies": {
-    "@itwin/itwinui-icons-react": "^1.8.0",
-    "@itwin/itwinui-react": "^1.40.1",
+    "@itwin/itwinui-icons-react": "^2.1.0",
+    "@itwin/itwinui-react": "^2.7.0",
+    "@itwin/itwinui-variables": "^2.0.0",
     "classnames": "^2.3.1",
     "i18next": "^10.2.2",
     "react-redux": "^7.2.0",
     "redux": "^4.0.3",
     "rxjs": "^6.6.2"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "~3.6.0",
-    "@itwin/appui-layout-react": "~3.6.0",
-    "@itwin/appui-react": "~3.6.0",
-    "@itwin/build-tools": "~3.6.0",
-    "@itwin/components-react": "~3.6.0",
-    "@itwin/core-bentley": "~3.6.0",
-    "@itwin/core-common": "~3.6.0",
-    "@itwin/core-frontend": "~3.6.0",
-    "@itwin/core-geometry": "~3.6.0",
-    "@itwin/core-i18n": "~3.6.0",
-    "@itwin/core-markup": "~3.6.0",
-    "@itwin/core-orbitgt": "~3.6.0",
-    "@itwin/core-quantity": "~3.6.0",
-    "@itwin/core-react": "~3.6.0",
-    "@itwin/eslint-plugin": "~3.6.0",
-    "@itwin/imodel-components-react": "~3.6.0",
-    "@itwin/presentation-common": "~3.6.0",
-    "@itwin/presentation-components": "~3.6.0",
-    "@itwin/presentation-frontend": "~3.6.0",
-    "@itwin/presentation-testing": "~3.6.0",
-    "@itwin/webgl-compatibility": "~3.6.0",
+    "@itwin/appui-abstract": "3.7.0-dev.6",
+    "@itwin/appui-layout-react": "4.0.0-dev.12",
+    "@itwin/appui-react": "4.0.0-dev.12",
+    "@itwin/build-tools": "3.7.0-dev.6",
+    "@itwin/components-react": "4.0.0-dev.12",
+    "@itwin/core-backend" : "3.7.0-dev.6",
```

Comment:
Why does tree-widget needs dependencies on `core-backend` and `presentation-backend`? Also I think it would be good to update `devDependencies` to match `peerDependencies` versions.

Reply:
Tree-widget needs dependencies on core-backend and presentation-backend because [@itwin/presentation-testing](https://github.com/iTwin/presentation/blob/master/packages/testing/package.json#L40) has them as peer dependencies.